Library/Formula/gcviewer.rb

```diff
@@ -0,0 +1,14 @@
+class Gcviewer < Formula
+  homepage "https://github.com/chewiebug/GCViewer"
+  version "1.34.1"
+  url "https://github.com/chewiebug/GCViewer/archive/#{version}.tar.gz"
+  sha256 "e0e97a94c80be8323772dc8953d463fe8eaf60c3a1c5f212d0b575a3b5c640ba"
+
+  depends_on "maven" => :build
```

Comment:
You *might* need a `depends_on :java` here as well. Just ensures the right `JAVA_HOME` is set.